pom.xml

```diff
@@ -358,6 +359,12 @@
                 <version>${postgresql.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc8</artifactId>
+                <version>${oracle.version}</version>
```

Comment:
you need add scope `<scope>test</scope>`

Reply:
ok. i will fix it and explain in the document